packages/product-catalogue/src/components/ProductForm/index.tsx

```diff
@@ -182,7 +184,11 @@ const ProductForm = (props: ProductFormProps) => {
             setImageUrl(url);
           }
         })
-        .catch((_: HTTPError) => sendErrorNotification(lang.ERROR_IMAGE_LOAD));
+        .catch((_: HTTPError) => {
+          setApiError(true);
+          sendErrorNotification(lang.ERROR_IMAGE_LOAD);
+        })
+        .finally(() => setIsLoading(false));
```

Comment:
should this async function affect the whole form?

Reply:
it does in a way it will show spinner loader till the data is loaded and once data is loaded the actual form will appear not like old time when form used to show even if the image was not loaded